salt/modules/aptpkg.py

```diff
@@ -158,6 +159,24 @@ def _check_apt():
         )
 
 
+def _call_apt(args, scope=True, **kwargs):
+    '''
+    Call apt* utilities.
+    '''
+    cmd = []
+    if scope and salt.utils.systemd.has_scope(__context__) and __salt__['config.get']('systemd.scope', True):
+        cmd.extend(['systemd-run', '--scope'])
+    cmd.extend(args)
+
+    params = {'output_loglevel': 'trace',
+              'ignore_retcode': True,
```

Comment:
It's rare that we need to do ignore the retcode, I don't think this should be the default. Why not just remove this line and pass through `ignore_retcode=True` in the kwargs in the two places where we use it (`autoremove` and `list_repo_pkgs`)?

Reply:
Yes, that's would do, of course.